api/dataprocessor.go

```diff
@@ -188,6 +188,8 @@ func (s *Server) getTargets(reqs []models.Req) ([]models.Series, error) {
 	if len(localReqs) > 0 {
 		wg.Add(1)
 		go func() {
+			// the only errors returned are from us catching panics, so we should treat them
+			// all as internalServerErrors
```

Comment:
I don't agree with this, but anyway we can revisit this later, it's not important enough to block this PR. However let's make the log prefixes consistent.